### PR TITLE
JSON summary output

### PR DESCRIPTION
**Pull request by jakub-polak**
Introducing JSON format output - `chia farm summary --json` or `chia farm summary -j`

It would be very handy if you want to integrate any chia monitoring based on HTTP requests

### CODE REVIEW

File: chia/cmds/farm_funcs.py [hunk 1]
```diff
@@ -181,69 +181,81 @@ async def challenges(farmer_rpc_port: int, limit: int) -> None:
...
+async def summary(
+        rpc_port: int, wallet_rpc_port: int, harvester_rpc_port: int, farmer_rpc_port: int, json_format: bool
+) -> None:
     amounts = await get_wallets_stats(wallet_rpc_port)
     plots = await get_plots(harvester_rpc_port)
     blockchain_state = await get_blockchain_state(rpc_port)
     farmer_running = await is_farmer_running(farmer_rpc_port)
 
-    print("Farming status: ", end="")
+    summary = {}
```
Comment (Nikolaj-K) [inline chia/cmds/farm_funcs.py:192]: Python is probably forgiving but I'm not sure that this should be called the same as the function using it.

Comment (jakub-polak) [reply chia/cmds/farm_funcs.py:192]: Sure, that makes sense. Anyway - naming is the hardest part of software engineering, so having in mind that this function is probably never going to be called recursively shadowing its name should be fine as well.

Comment (johnbiggs) [reply chia/cmds/farm_funcs.py]: Everything else uses the word show.

Comment (jakub-polak) [reply chia/cmds/farm_funcs.py]: Fair point, I'll change the name of function to `show_summary`

**Pull request closed without merging**